framework/include/base/MooseObject.h

```diff
@@ -19,7 +19,8 @@
 
 #define usingMooseObjectMembers                                                                    \
   using MooseObject::isParamValid;                                                                 \
-  using MooseObject::paramError
+  using MooseObject::paramError;                                                                   \
+  using MooseObject::getParam
```

Comment:
Remove this since doing this doesn't introduce `getParam` as a template name, which it must always be used as